docs/release_notes/v1.11.1.md

```diff
@@ -0,0 +1,113 @@
+# Dapr 1.11.1
+
+## Fixed an issue with Service Invocation on Kubernetes clusters with the permission enforcement admission controller enabled.
+
+### Problem
+
+Updating Dapr from 1.10 to 1.11 on Kubernetes clusters gives the following error
+when using Service Invocation:
+
+```
+unable to create Dapr service for wrapper, service: test/test-dapr, err:
+services \"test-dapr\" is forbidden: cannot set blockOwnerDeletion if an
+ownerReference refers to a resource you can't set finalizers on:
+```
+
+### Impact
+
+Since v1.11.0 Service Invocation on a subset of Kubernetes clusters is unusable.
+
+### Root cause
+
+When using the permission enforcement admission controller on Kubernetes,
+Service Invocation fails because the operator is unable to create or update
+Services. This is particularly problematic when for OpenShift clusters as they
+have this admission controller enabled by default.
+
+### Solution
+
+Give the operator RBAC permissions to update Deployment and StatefulSet
+finalizers, as well as delete services.
+
+## Fixed a Goroutine leak in Actor reminders and timers.
+
+### Problem
+
+The Daprd sidecar process would increase in memory over time when using Actor
+reminders and timers.
+
+### Impact
+
+Using Actor reminders and timers in v1.11.0, Daprd's memory consumption would
+continuously grow overtime, leading to OOM kills or underutilized nodes.
```

Comment:
```suggestion
Using Actor reminders and timers in Dapr 1.11.0 and earlier could cause a goroutine leak that would grow over time.
This could cause increase CPU utilization and/or memory exhaustion.
```